Makefile

```diff
@@ -1,23 +1,24 @@
+app_dir='browser'
 
 .PHONY: default
 default: makedb
 
 .PHONY: cleanpositionthumbs
 cleanpositionthumbs:
-	find . -name '*thumbnail*.gif' | xargs rm
+	find $(app_dir) -name '*thumbnail*.gif' | xargs rm
 
 .PHONY: cleanthumbs
 cleanthumbs: cleanpositionthumbs
-	find . -name '*thumbnail*.png' | xargs rm
-	rm -rf protected/images
+	find $(app_dir) -name '*thumbnail*.png' | xargs rm
+	rm -rf $(app_dir)/protected/images
```

Comment:
Actually, I'll be more comfortable with this if this line read:

```
rm -rf -- "$(app_dir)/protected/images"
```

with the `--` to indicate end of options and the quotes to allow app_dir to have whitespace in it.